packages/UI/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "noEmit": false,
     "composite": false,
     "incremental": true,
+    "outDir": "dist",
```

Comment:
@jeyanthi-sm @ajaywayase always make sure that code owners review your changes before merging.

Please delete the outDir.